docsite/source/plugins.html.md

```diff
@@ -22,6 +22,77 @@ App[:logger]
 App.logger           
 ```
 
+## Zeitwerk
+
+With `:zeitwerk` plugin you can easily use [Zeitwerk](https://github.com/fxn/zeitwerk) as your applications's code loader.
+
+> Given a conventional file structure, Zeitwerk is able to load your project's classes and modules on demand (autoloading), or upfront (eager loading). You don't need to write require calls for your own files, rather, you can streamline your programming knowing that your classes and modules are available everywhere. This feature is efficient, thread-safe, and matches Ruby's semantics for constants. (Zeitwerk docs)
+
+### Example
+
+Here is an example of using Zeitwerk plugin:
+
+```ruby
+class App < Dry::System::Container
+  use :env, inferrer: -> { ENV.fetch("RACK_ENV", :development).to_sym }
+  use :zeitwerk
+
+  configure do |config|
+    config.component_dirs.add "lib"
+  end
+end
+```
+
+For a more in depth and runnable example, [click here](https://github.com/dry-rb/dry-system/tree/master/examples/standalone).
+
+### Inflections 
+
+The plugin tries to handle most of the plumbing for you. For example, is uses the container's own inflector to resolve constant names. So if Zeitwerk is having trouble resolving some constants, just update the container's inflector like so:
+
+```ruby
+class App < Dry::System::Container
+  use :zeitwerk 
+
+  configure do |config|
+    config.inflector = Dry::Inflector.new do |inflections|
+      inflections.acronym('API')
+    end
+
+    # ...
+  end
+end
+```
+
+### Eager Loading
+
+By default, the plugin will have zeitwerk eager load when using the `:env` plugin sets the environment to `:production`. However, you can change this behavior by passing `:eager_load` option to the plugin:
+
+```ruby
+class App < Dry::System::Container
+  use :zeitwerk, eager_load: true
+end
+```
+
+### Debugging
+
+When you are developing your application, you can enable the plugin's debugging mode by passing `:debug` option to the plugin:
```

Comment:
It would be great to see an explanation of what the debug mode means/does.

Reply:
It's nothing fancy. Added some extra text to make it clearer :)